.pre-commit-config.yaml

```diff
@@ -50,6 +50,15 @@ repos:
     rev: v0.761
     hooks:
     -   id: mypy
```

Comment:
Why are these changes necessary?

Reply:
Unfortunately mypy does not support having twice the same module name ([failing lint task](https://community-tc.services.mozilla.com/tasks/JCk2ea2TStq4FDguMjZD6Q/runs/0/logs/https%3A%2F%2Fcommunity-tc.services.mozilla.com%2Fapi%2Fqueue%2Fv1%2Ftask%2FJCk2ea2TStq4FDguMjZD6Q%2Fruns%2F0%2Fartifacts%2Fpublic%2Flogs%2Flive.log#L110))

Reply:
Painful :(